ICAFIX/scripts/icaDim.m

```diff
@@ -19,71 +19,74 @@
 
 %%%%%%%%%%%%%%%%%%%%
 
-OrigdataSizeOne=size(Origdata,1);
-OrigdataSizeTwo=size(Origdata,2);
-OrigdataMask=std(Origdata,[],2)>0;
+Nvox = size(data,1);
+Ntp = size(data,2);
+
+% Mask of non-zero voxels
+mask = std(data,[],2)>0;
+% Apply mask, reusing 'data' variable for memory efficiency
+data = data(mask,:);
```

Comment:
Input variables don't count as used memory until they are assigned to, nor does a trivial copy (no indexing expression).  I prefer not overwriting variables with modified versions, and in this case and data_detrend, it shouldn't improve memory usage versus my version.

Reply:
I'll add the `usemask` stuff, but I don't see the disadvantage of overwriting variables with a modified version, esp. since it is a misnomer to name the variable `data_detrend` since whether it is actually detrended depends on the value of `DEMDT`.